web/src/components/layout/header/ActivePipelines.vue

```diff
@@ -1,58 +1,57 @@
 <template>
   <IconButton :title="$t('pipeline_feed')" class="!p-1.5 relative text-current active-pipelines-toggle" @click="toggle">
-    <div v-if="activePipelines.length > 0" class="spinner m-1">
-      <div class="spinner-ring ring1" />
-      <div class="spinner-ring ring2" />
-      <div class="spinner-ring ring3" />
-      <div class="spinner-ring ring4" />
-    </div>
+    <div v-if="activePipelines.length > 0" class="spinner" />
     <div
-      class="flex items-center justify-center h-full w-full font-bold bg-white bg-opacity-15 dark:bg-black dark:bg-opacity-10 rounded-md"
+      class="z-1 flex items-center justify-center h-full w-full font-bold bg-white bg-opacity-15 dark:bg-black dark:bg-opacity-10 rounded-md"
     >
-      {{ activePipelines.length || 0 }}
+      <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
+      {{ pipelineCount > 9 ? '9+' : pipelineCount }}
```

Comment:
the option to know the exact nuber would be still nice ...

what about a mouse hover over to show the number?

Reply:
Good idea will add it

Reply:
Good idea will add it

Reply:
![image](https://github.com/user-attachments/assets/0c422552-713a-47ab-8f31-008a7ad56495)